docs/events.md

```diff
@@ -184,22 +128,38 @@ Event that represents a device sending its config to the services that are inter
   - `id` **String** thing's ID
   - `config` **Array** config items, each one formed by:
     - `sensorId` **Number** sensor ID
-    - `change` **Boolean** enable sending sensor data when its value changes
-    - `timeSec` **Number** - **Optional** time interval in seconds that indicates when data must be sent to the cloud
-    - `lowerThreshold` **Optional (Depends on schema's valueType)** send data to the cloud if it's lower than this threshold
-    - `upperThreshold` **Optional (Depends on schema's valueType)** send data to the cloud if it's upper than this threshold
+    - `schema` **JSON Object** schema item, each one formed by:
+      - `typeId` **Number** semantic value type (voltage, current, temperature, etc)
+      - `valueType` **Number** data value type (boolean, integer, etc)
+      - `unit` **Number** sensor unit (V, A, W, etc)
+      - `name` **String** sensor name
+    - `event` **JSON Object** event item, each one formed by:
+      - `change` **Boolean** enable sending sensor data when its value changes
+      - `timeSec` **Number** - **Optional** time interval in seconds that indicates when data must be sent to the cloud
+      - `lowerThreshold` **(Depends on schema's valueType)** - **Optional** send data to the cloud if it's lower than this threshold
+      - `upperThreshold` **(Depends on schema's valueType)** - **Optional** send data to the cloud if it's upper than this threshold
+
+  The semantic specification that defines `valueType`, `unit` and `typeId` properties can be find [here](https://knot-devel.cesar.org.br/doc/thing/unit-type-value.html).
 
   Example:
 
   ```json
   {
     "id": "fbe64efa6c7f717e",
-    "config": [{
+    "config" : [{
       "sensorId": 1,
-      "change": true,
-      "timeSec": 10,
-      "lowerThreshold": 1000,
-      "upperThreshold": 3000
+      "schema": {
+        "valueType": 0xFFF1,
+        "unit": 0,
+        "typeId": 3,
+        "name": "Door lock",
+      },
```

Comment:
Can you check if `valueType` and `typeId` values are swapped? I'm a bit outdated on some recent changes, so please LMK if this was swapped intentionally (see [KNoT Devel](https://knot-devel.cesar.org.br/doc/thing/unit-type-value.html) and the [previous commit](https://github.com/CESARBR/knot-babeltower/pull/91/commits/4746d30ecde7822dd0c1b16fd3970dc8fd49bf46#diff-833b6d7070c26b0f7548f4ffdc70eb8eL145)). :eyes: 
>**_note_**: in case the swap wasn't intentional, make sure to update other occurrences :grimacing: 

```suggestion
      "schema": {
        "valueType": 0xFFF1,
        "unit": 0,
        "typeId": 3,
        "name": "Door lock",
      },
```

Reply:
Fixed